src/nav/apm.yml

```diff
@@ -1186,19 +1212,6 @@ pages:
                 path: /docs/apm/agents/ruby-agent/api-guides/sending-handled-errors-new-relic
               - title: Third party instrumentation
                 path: /docs/apm/agents/ruby-agent/api-guides/third-party-instrumentation
-          - title: Background jobs
-            path: /docs/apm/agents/ruby-agent/background-jobs
-            pages:
-              - title: Resque instrumentation
-                path: /docs/apm/agents/ruby-agent/background-jobs/resque-instrumentation
-              - title: Sidekiq instrumentation
-                path: /docs/apm/agents/ruby-agent/background-jobs/sidekiq-instrumentation
-              - title: Rake instrumentation
-                path: /docs/apm/agents/ruby-agent/background-jobs/rake-instrumentation
-              - title: Monitor background processes
-                path: /docs/apm/agents/ruby-agent/background-jobs/monitor-ruby-background-processes
-              - title: 'Delayed::Job instrumentation'
-                path: /docs/apm/agents/ruby-agent/background-jobs/delayedjob-instrumentation
           - title: Attributes
```

Comment:
As long as it is okay with the other agent teams, I think the document in this section could be moved to "Features"

Reply:
done!